modules/Makefile.docker

```diff
@@ -49,14 +48,23 @@ DOCKER_BIND_PORT ?= $(RANDOM_PORT):80
 DOCKER_SHELL ?= /bin/bash
 
 # Docker client 
-DOCKER_CMD ?= $(MAKEFILE_DIR)/bin/docker-$(DOCKER_VERSION)
+DOCKER_CMD ?= $(shell which docker)
 
 # Arguments passed to "docker build"
 DOCKER_BUILD_OPTS ?=
 
 .PHONY : docker\:build docker\:push docker\:pull docker\:clean docker\:run docker\:shell docker\:attach docker\:update docker\:start docker\:stop docker\:rm docker\:logs 
 
+deps::
+	$(call assert_set DOCKER_CMD)
+	$(call assert_set DOCKER_HOST)
```

Comment:
I thought this was docker-machine specific? Don't think DOCKER_HOST and DOCKER_CERT_PATH needed for native docker.